src/data/decks.json

```diff
@@ -726,5 +726,12 @@
     "faction": "ironclad",
     "author": "Helios",
     "category": "EQUALS"
+  },
+  {
+    "name": "Cancerous Canon",
+    "id": "NUkxLDVJMiw1TjMsNUk0LDVJOCw1STksNUk2LDVJMTUsNUkyMCw1STEyLDVJMTksNUkyOA",
+    "faction": "ironclad",
+    "author": "LightingLords",
```

Comment:
Isn’t it from Frrostkhan (as the title suggests)?

Reply:
Frostkhan's offers his viewers to test their decks with his lvl 5 cards in D1. Since he credits them (see https://www.youtube.com/watch?v=1z6ezlakM7g), I think it would be fair to keep LightingLords as the author